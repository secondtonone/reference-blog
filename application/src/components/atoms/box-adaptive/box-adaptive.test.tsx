/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import BoxAdaptive from '.';

describe('Match snapshots BoxAdaptive', () => {
  it('default', () => {
    const { asFragment } = render(<BoxAdaptive />, {});
    expect(asFragment()).toMatchSnapshot();
  });
  it('with children', () => {
    const { asFragment } = render((
      <BoxAdaptive>
        <div className="children" />
      </BoxAdaptive>
    ), {});
    expect(asFragment()).toMatchSnapshot();
  });
});
