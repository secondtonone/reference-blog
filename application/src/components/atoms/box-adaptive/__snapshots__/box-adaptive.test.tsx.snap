// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots BoxAdaptive default 1`] = `
<DocumentFragment>
  <div
    class="box-adaptive__BoxStyled-sc-g2wy39-0"
  />
</DocumentFragment>
`;

exports[`Match snapshots BoxAdaptive with children 1`] = `
<DocumentFragment>
  <div
    class="box-adaptive__BoxStyled-sc-g2wy39-0"
  >
    <div
      class="children"
    />
  </div>
</DocumentFragment>
`;
